New introduction (#132)
web-platform-tests/interop: 2023/README.md

# Interop 2023

> Interop 2023 is an effort to increase interoperability across browsers in key technical areas that are of high priority to web developers and for user needs. It's part of the [web-platform-tests](https://github.com/web-platform-tests/wpt) (WPT) project — an automated test suite for web standards — and run by a team of representatives from companies that make substantial contributions to browser rendering engines.

Reviewer: ```suggestion
Interop 2023 is an effort to increase interoperability across browsers in key technical areas that are of high priority to web developers and end users. It's part of the [web-platform-tests](https://github.com/web-platform-tests/wpt) (WPT) project — an automated test suite for web standards — and run by a team of representatives from companies that make substantial contributions to browser rendering engines.
```

Similar to [Interop 2022](https://wpt.fyi/interop-2022), the tests selected will be continuously run on automated testing infrastructure. The test pass rates for each browser rendering engine will be displayed on the Interop 2023 Dashboard — displaying the percentage of passing tests in each chosen area, and an overall total score. The dashboard will also display scores for group progress on particular Investigation Efforts selected for the the Interop team to work on throughout the year.

> Interop 2023 is tightly focused on technology that is already specified in web standards. This is not a place to invent new web technology, nor solve complex problems that need web standards. That work should happen in the standards bodies. 

Reviewer: ```suggestion
Interop 2023 is tightly focused on technology that is already specified in web standards. This is not a place to invent new web technology: that work should happen in the standards bodies. 
```
Or I'm not sure quite what the second clause was trying to say that's different from the first. Maybe it was trying to make the distinction between really novel standards and major (non-interop motivated) changes to existing standards? If so perhaps "This is not a place to invent new web technology, nor make novel additions to existing specifications." (there are some edge cases there with investigate, but it probably doesn't matter).

Reviewer 2: I like the simplification of the first clause. Did you mean to suggest a semicolon though? IIRC you suggested it in the other place:

> Interop 2023 is not a venue for specifying new features; that work happens in working groups within organizations such as W3C and WHATWG.

Reviewer: Sure, a semicolon works.

# Interop 2023

> Interop 2023 is a project of Web Platform Tests (WPT). Used by many browser engineering teams, [WPT](https://web-platform-tests.org/) is an automated [test suite](https://github.com/web-platform-tests/wpt) for the web standards created within the WHATWG, W3C, and other standards organizations. 

Reviewer 2: ```suggestion
Interop 2023 is an effort to increase interoperability across browsers in key technical areas that are of high priority to web developers and for user needs. It's part of the [web-platform-tests](https://github.com/web-platform-tests/wpt) (WPT) project, an automated test suite for web standards, used by many browser engineering teams.
```

> The aim of the annual Interop project is to increase interoperability across browsers in key technical areas that are of high priority to web developers and for user needs. Interop is run by a steering committee of representatives from the companies that make substantial contributions to browser rendering engines. 

Reviewer 2: ```suggestion
Interop is run by a team of representatives from the companies that make substantial contributions to browser rendering engines.
```

> Similar to [Interop 2022](https://wpt.fyi/interop-2022), the tests selected are continuously run on WPT’s automated testing infrastructure. The test pass rates for each browser rendering engine are displayed on the Interop Dashboard — displaying the percentage of passing tests in each chosen area, and an overall total score. The Interop Dashboard also displays scores for group progress on particular Investigation Projects selected for the the Interop team to work on throughout the year.

Reviewer 2: ```suggestion
Similar to [Interop 2022](https://wpt.fyi/interop-2022), the tests selected are continuously run on automated testing infrastructure. The test pass rates for each browser rendering engine will be displayed on the Interop 2023 Dashboard — displaying the percentage of passing tests in each chosen area, and an overall total score. The dashboard will also display scores for group progress on particular Investigation Efforts selected for the the Interop team to work on throughout the year.
```

> Interop is tightly focused on 1) technology that can be tested on WPT’s current infrastructure; and 2) technology that is already specified on web standards. This is not a place to invent new web technology, or solve complex problems that need web standards. That works should happen in the standards bodies. 

Reviewer 2: ```suggestion
Interop 2023 is tightly focused on technology that is already specified in web standards. This is not a place to invent new web technology, nor solve complex problems that need web standards. That work should happen in the standards bodies. 
```
